### PR TITLE
Update HarvardDataVerse to release 2.0.0

### PR DESCRIPTION
**Pull request by zamattiac**

### CODE REVIEW

File: providers/edu/harvarddataverse/normalizer.py [hunk 1]
```diff
@@ -9,33 +9,21 @@ class Person(Parser):
...
     description = Try(ctx.description)
-    contributors = Map(Delegate(Contributor), Try(ctx.authors))
     date_published = ParseDate(ctx.published_at)
-    links = Concat(
-        Delegate(ThroughLinks, ctx.url),
-        Delegate(ThroughLinks, ctx.image_url),
-    )
+    identifiers = Map(Delegate(WorkIdentifier), ctx.url)
+
+    related_agents = Map(Delegate(Person), ctx.authors)
```
Comment (aaxelb) [inline providers/edu/harvarddataverse/normalizer.py]: Should delegate through `Creator`.

File: providers/edu/harvarddataverse/normalizer.py [hunk 2]
```diff
@@ -9,33 +9,21 @@ class Person(Parser):
 
 
 class Contributor(Parser):
```
Comment (aaxelb) [inline providers/edu/harvarddataverse/normalizer.py]: For authors, this should be `Creator`, and should have `cited_as` and `order_cited` fields.

**Comment by zamattiac** [comment]
@aaxelb which throughtables am I missing besides Creator?

**Comment by aaxelb** [comment]
Just Creator, really. I was redundantly summarizing my comments with vague, general advice 👍

File: providers/edu/harvarddataverse/normalizer.py [hunk 3]
```diff
@@ -8,34 +8,24 @@ class Person(Parser):
...
+    schema = 'dataset'
     title = ctx.name
     description = Try(ctx.description)
-    contributors = Map(Delegate(Contributor), Try(ctx.authors))
     date_published = ParseDate(ctx.published_at)
-    links = Concat(
-        Delegate(ThroughLinks, ctx.url),
-        Delegate(ThroughLinks, ctx.image_url),
-    )
+    identifiers = Map(Delegate(WorkIdentifier), ctx.url)
```
Comment (chrisseto) [inline providers/edu/harvarddataverse/normalizer.py]: Any reason to not keep the image url?

File: providers/edu/harvarddataverse/normalizer.py [hunk 4]
```diff
@@ -8,34 +8,24 @@ class Person(Parser):
     suffix = ParseName(ctx).suffix
 
 
-class Contributor(Parser):
-    person = Delegate(Person, ctx)
-    cited_name = ctx
+class Creator(Parser):
+    agent = Delegate(Person, ctx)
     order_cited = ctx('index')
+    cited_as = ParseName(ctx).last
```
Comment (chrisseto) [inline providers/edu/harvarddataverse/normalizer.py]: `cited_as` should be the original representation of that persons name.

File: providers/edu/harvarddataverse/normalizer.py (hunk 3 above)
Comment (chrisseto) [inline providers/edu/harvarddataverse/normalizer.py]: Throw `global_id` in as well, just for good measure. It will be de-duped if necessary

Comment (zamattiac) [inline providers/edu/harvarddataverse/normalizer.py]: I wasn't able to access any of the image urls, so I didn't know if they were unique identifiers or what they even were

**Pull request merged**